Weak scaling in benchmark example (#899)
openPMD/openPMD-api: docs/source/usage/benchmarks.rst

Benchmarks
==========

Parallel benchmark 8
--------------------

> Build based on the helper functions in the :ref:`benchmark utilities <utilities-benchmark>`_, this benchmark executes a simple parallel read-write test.

Reviewer: ```suggestion
Build based on the helper functions in the :ref:`benchmark utilities <utilities-benchmark>`, this benchmark executes a simple parallel read-write test.
```

> In particular, this test case writes and reads a 4D array of type ..., sliced 1D along the ... dimension.

Reviewer: @franzpoeschel can you potentially fill in the `...`s? :)

> In particular, this test case writes and reads a 4D array of type ``long``, sliced 1D along the first dimension.

Reviewer: ```suggestion
In particular, this test case writes and reads a 4D array of type ``uint64_t``, sliced 1D along the first dimension.
```

.. code-block:: cpp

   openPMD::Extent total{

>        100 * scale_up, // slices along or this...?

Reviewer: let's place the `// sliced along this axis over MPI ranks` comment on the right axis

>        100 * scale_up, // slices along or this axis?

Reviewer: ```suggestion
       100 * scale_up, // sliced along this axis over MPI ranks
```

       100,
       100,
       10 // or this axis?
   };

> That means in the strong-scaling case, always ... GB of data are produced.

Reviewer: Here as well please :)
